src/Regmel/Controller/Web/RegisterController.php

```diff
@@ -43,7 +43,7 @@ public function registerCity(Request $request): Response
     {
         $snpEmail = $this->parameterBag->get('app.email.address');
 
-        $maxFileSize = $this->getParameter('max_file_size');
```

Comment:
Sara, o certo é com essa linha mesmo viu, se não tiver funcionando é por causa do metodo `$this->getParameter()`